gradle.properties

```diff
@@ -1,2 +1,2 @@
 #path to Wowza home directory. Do not use qoutations marks.
-WMSINSTALL_HOME = /opt/local/WowzaStreamingEngine
\ No newline at end of file
+WMSINSTALL_HOME = C:\\Program Files (x86)\\Wowza Media Systems\\Wowza Streaming Engine 4.0.1
```

Comment:
I think it's best if you don't check this in. I prefer gradle not to be couple with a specific Wowza version.
Besides, if you're in windows you already have %WMSINSTALL_HOME% created by Wowza